core/ledger/kvledger/upgrade_dbs_test.go

```diff
@@ -62,22 +54,104 @@ func TestUpgradeDBs(t *testing.T) {
 	require.Equal(t, os.IsNotExist(err), true)
 
 	// upgrade again should be successful
-	err = UpgradeDBs(rootFSPath)
+	err = UpgradeDBs(conf)
 	require.NoError(t, err)
 }
 
-func TestUpgradeIDStoreWrongFormat(t *testing.T) {
+func TestUpgradeDBsWithV13CouchDB(t *testing.T) {
```

Comment:
For rollback and reset commands, we have added tests in the ledger-level integration test pkg. This is because these commands modify blockstore and many DBs (affects multiple sub-packages in the ledger). The upgradeDB command is also modifying blockstore and manyDBs. Further, `TestUpgradeDBsWithV13CouchDB()` and `TestUpgradeDBsWithV13CouchDB()` access testdata present in the ledger-level integration test pkg. Hence, I would prefer moving these tests to `kvledger/tests` to maintain consistency. 

This reminds me of the discussion @manish-sethi and I had a year back on moving recovery-related tests to `kvledger/tests` pkg as it involves multiple sub-packages in the ledger.

Reply:
Thank you, Senthil. Agree that we should have ledger-level integration tests for UpgradeDBs under kvledger/tests. However, these tests are meant for unit tests to verify idStore is updated correctly and databases are dropped or not dropped as expected.

Reply:
In `kvledger/tests`, we do have [verifyRebuilableDoesNotExist()](https://github.com/hyperledger/fabric/blob/master/core/ledger/kvledger/tests/env.go#L144) and [verifyRebuilablesExist()](https://github.com/hyperledger/fabric/blob/master/core/ledger/kvledger/tests/env.go#L126) which checks whether the DBs has been dropped or DBs exist. We use these in both reset and rollback tests. These functions can be extended to add a CouchDB check too. You can find no UT for rollback and reset commands in `kvledger` package. 

As I mentioned in one of the above comments, if we have `hasVersion20Format()`, we can also add an exported function to `test_export.go` and use it in the `kvledger/tests` to check whether the format has been applied.

We can discuss further and can be done in a separate PR too.